plugin/helpers.go

```diff
@@ -77,4 +79,19 @@ func TrackTime(start time.Time, name string) {
 // Attributes returns the entry's attribtues
 func Attributes(e Entry) EntryAttributes {
 	return e.attributes()
-}
\ No newline at end of file
+}
+
+var isInteractive bool = (isatty.IsTerminal(os.Stdin.Fd()) || isatty.IsCygwinTerminal(os.Stdin.Fd())) &&
+	(isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd()))
+
+// InitInteractive is used by Wash commands to set option-specific overrides. Only sets
+// interactivity to true if it already was and 'init' is also true.
+func InitInteractive(init bool) {
+	isInteractive = init && isInteractive
+}
+
+// IsInteractive returns true if Wash is running as an interactive session. If false, please don't
+// prompt for input on stdin.
+func IsInteractive() bool {
```

Comment:
How about `plugin.Prompt(msg) (string, error)` ? That encapsulates all of this interactivity stuff, which lets us say "Use plugin.Prompt whenever you want to prompt something from the user."

Reply:
Hmm, could add that. I still want the IsInteractive call for cmd to avoid duplicating it.